harness/testTypedArray.js

```diff
@@ -20,11 +20,22 @@ var typedArrayConstructors = [
   Uint8ClampedArray
 ];
 
+var numericTypedArrayConstructors = typedArrayConstructors.slice();
```

Comment:
Once BigInt reaches Stage 4, Test262 will need to decide if it wants to start making implementations fail (where they previously passed) by adding BigInt64Array and BigUint64Array directly (unconditionally) to the `typedArrayConstructors` array.

Reply:
Yes, I'm reviewing this branch and checking the modified files. I'm gonna do some work here to split tests so we keep tests for TypedArrays going as ok while implementations for BigInt can run separately.